### ISSUE TITLE
Header Order <h6> without <h5>

### ISSUE THREAD

**Issue by JohnHewittCriterion** [flexion/fs-intake-module#766]
Headers are used by Screen readers and other tools to create a table of contents for users so they can navigate through the pages efficiently.

Link: https://fs-intake-staging.app.cloud.gov/applications/temp-outfitters/new
![image](https://user-images.githubusercontent.com/28604890/29431759-1f30fa46-8367-11e7-99d9-aa4043e23e17.png)

Headers must be consecutive when increasing.  This usually fails to happen when headers are being used for visual styles instead of document structure.

WCAG: 1.3 Adaptable - 1.3.1 Info and Relationship 
Linkage: EPERM-69

**Comment by npage123** [comment]
- [ ] Check with Mac Voice Over
- [ ] Check with JAWS

**Comment by sethalt** [comment]
- [x] Check with Mac Voice Over
- [x] Check with JAWS
